src/libmongoc/src/mongoc/mongoc-error-private.h

```diff
@@ -72,7 +74,7 @@ void
 _mongoc_write_error_handle_labels (bool cmd_ret,
                                    const bson_error_t *cmd_err,
                                    bson_t *reply,
-                                   int32_t server_max_wire_version);
+                                   mongoc_server_description_t *sd);
```

Comment:
@kevinAlbs @kkloberdanz: I just thought of this after merging (of course), but would you have preferred I use `const mongoc_server_description_t *sd` here or does it make no difference?

Reply:
I would have a slight preference to making it `const`, but not a hard requirement in my book.